src/scripts/shell/platform-choice/platform-choice.component.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
```

Comment:
looks like lodash was added here:
https://github.com/DestinyItemManager/DIM/commit/18f4f50d8b0fc24eb963363f01ddd0da4acda4c6#diff-2018087f584c4398b5c3a23fc0e5f9dbR9

but hasn't been used anywhere. do we want to migrate to it eventually?